datalad/cmdline/helpers.py

```diff
@@ -313,6 +313,10 @@ def strip_arg_from_argv(args, value, opt_names):
 
 
 def run_via_pbs(args, pbs):
+    warnings.warn("Job submission via --pbs-runner is deprecated."
+                  "Use something like condor_run",
```

Comment:
```suggestion
                  "Use something like condor_run from datalad-htcondor extension",
```

Reply:
Err, no. I meant the standard HTCondor command. I do not want to advertise `datalad-htcondor`.

Reply:
ah, misunderstood. ok